app/_src/gateway-operator/get-started/konnect/horizontal-autoscaling.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Horizontally autoscale a Data Plane
+content-type: tutorial
+book: kgo-konnect-get-started
+chapter: 4
+---
+
+{% if_version gte:1.2.0 %}
```

Comment:
Not sure if this is necessary given that this page will only be rendered in 1.2.x and up due to not being included in older nav files.

Technically ( I can be proven wrong ) this is not necessary given the above so I'll defer to @Kong/docs-maintainers if this makes sense.

Reply:
This is correct. You can remove the `if_version` as it's only included in 1.2.x+ nav files

Reply:
Gotcha, remove in this case. I was worried that users could still tinker with the version in the URL and get the site for earlier versions.